test/cpp/interop/interop_client.cc

```diff
@@ -83,7 +83,7 @@ CompressionType GetInteropCompressionTypeFromCompressionAlgorithm(
 }  // namespace
 
 InteropClient::InteropClient(std::shared_ptr<Channel> channel)
-    : channel_(channel) {}
+    : channel_(channel), stub_(TestService::NewStub(channel)) {}
```

Comment:
Thanks for calling this out!. I spoke to Yang and looks like this is intentional - to keep the test cases clean. For interop tests this makes sense but for the stress test use case, it is not a good idea to create a stub for every call :). I am going to make this configurable.